seatunnel-connectors-v2/connector-file/connector-file-sftp/src/main/java/org/apache/seatunnel/connectors/seatunnel/file/sftp/system/SFTPFileSystem.java

```diff
@@ -154,6 +155,29 @@ private boolean exists(ChannelSftp channel, Path file) throws IOException {
         }
     }
 
+    private String quote(String path) {
+        byte[] _path = path.getBytes(StandardCharsets.UTF_8);
+        int count = 0;
+        for (int i = 0; i < _path.length; i++) {
+            byte b = _path[i];
+            if (b == '\\' || b == '?' || b == '*') {
+                count++;
+            }
+        }
+        if (count == 0) {
+            return path;
+        }
+        byte[] _path2 = new byte[_path.length + count];
+        for (int i = 0, j = 0; i < _path.length; i++) {
+            byte b = _path[i];
+            if (b == '\\' || b == '?' || b == '*') {
+                _path2[j++] = '\\';
+            }
+            _path2[j++] = b;
+        }
+        return new String(_path2, 0, _path2.length, StandardCharsets.UTF_8);
+    }
```

Comment:
Please add a UT for this method.

Reply:
Hello, this is my first pr. Where should I add the test case for the individual method，thanks.

Reply:
put it in https://github.com/apache/seatunnel/tree/dev/seatunnel-connectors-v2/connector-file/connector-file-sftp/src/test/java/org/apache/seatunnel/connectors/seatunnel/file/sftp

Reply:
Thx, please review again.